docs/data-masking-comparison.md

```diff
@@ -0,0 +1,19 @@
+# Compare the data masking component to the data masking plugin
+
+Percona Server for MySQL 8.0.33, the data masking component adds the data masking component. Either the component or the plugin extends the server's functionality, but the architecture of a plugin is different than the component. Plugins have the following limitations:
+
+* Interact only with the server and do not interact with other plugins
+* Dependencies can be difficult to initialize 
+* Require a running server
+
+The component supports more arguments to the `gen_rnd_email()` function. The main differences between the component and plugin are the following:
+
+| Component | Plugin |
+|--- | --- |
+| Supports a service interface and loadable functions | Supports an interface only to loadable functions |
```

Comment:
we didn't implement the service interface

Reply:
removed line